src/solc_0.8/asset/Asset.sol

```diff
@@ -0,0 +1,10 @@
+//SPDX-License-Identifier: MIT
+// solhint-disable-next-line compiler-version
+pragma solidity 0.8.2;
+
+import "./ERC1155ERC721.sol";
+
+// solhint-disable-next-line no-empty-blocks
+contract Asset is ERC1155ERC721 {
```

Comment:
we should rename to something else otherwise we will need pass the full canonical name to get access to the artifact
We could call it AssetV2 ?

Reply:
yes! I've done this locally, just haven't pushed it yet.
I went with that exact name.